ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/router_path_selection.py

```diff
@@ -67,7 +67,16 @@ def _get_path_groups(self) -> list:
 
             keepalive = path_group.get("dps_keepalive_interval")
             if keepalive:
-                path_group_data["keepalive"] = keepalive
+                if keepalive.get("auto"):
+                    path_group_data["keepalive"] = {"auto": True}
+                else:
+                    interval = keepalive.get("interval")
+                    failure_threshold = keepalive.get("failure_threshold", 5)
+                    if interval:
```

Comment:
Can be dangerous if a value like 0 is supported. Maybe not in this case, but normally we do not rely on boolean checks for user inputs.

Reply:
Most places we check with `if interval is not None`

Reply:
Done. Removed the default 5 as well, I was using, couldn't decide on what description to use. 

Should an error be raised if only interval is configured/only failure_threshold is configured ?

Reply:
I think having a default for failure threshold makes sense, and how you had it before, it would only take effect if interval was given.

Reply:
Done.